android/app/src/main/kotlin/com/jabook/app/jabook/compose/feature/player/PlayerScreen.kt

```
@@ -2040,24 +2404,63 @@ private fun PlayerContent(
+                                }
+                                player.setOnErrorListener { _, _, _ ->
+                                    player.runCatching {
+                                        reset()
+                                        release()
+                                    }
+                                    bookmarkPlayer.value = null
+                                    isPlayingBookmarkAudio = false
+                                    true
+                                }
+                                player.prepareAsync()
+                            } catch (e: java.io.IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)